### DISCUSSION TITLE
VOLUME LEVEL ISSUE

### DISCUSSION THREAD

**Discussion by Rasb74** [rern/rAudio#871] [Bug Reports]
From the latest 3 updates still a volume level issue occur. 

I made a fresh install with the latest release 20220317 and still low volume output : 
1 - Beat perfect mode : Volume output very low compare to the previous releases
2 -without bitperfect and with mix device set : Volume level locked at 49. Impossible to go ahead

On the web interface it's possible to set the volume level at 100% but with no effect on the system. 
Alsamixer looks overload, as no action works on it. 
let's see the attached files
<img width="1131" alt="Capture d’écran 2022-04-06 à 18 45 47" src="https://user-images.githubusercontent.com/77485841/162026791-09518932-6536-4353-843b-1e6361b50e43.png">
<img width="560" alt="Capture d’écran 2022-04-06 à 18 45 23" src="https://user-images.githubusercontent.com/77485841/162026841-cf8512bd-5be7-4679-b330-1d25e017aac7.png">
![IMG_6829](https://user-images.githubusercontent.com/77485841/162029546-caa80780-f9d0-4779-971a-a5889da1102d.jpg)

**Comment by rern** [comment]
Try: Arrow right key to move current control, red text, to `<Digital>`

**Reply by Rasb74** [reply]
Already Tried, nothing works. I've made 3 fresh instals, with 3 different SD cards.
